<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>760</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="tabSpacing">
     <number>6</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1200</width>
       <height>682</height>
      </rect>
     </property>
     <attribute name="label">
      <string>待续</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>260</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1200</width>
       <height>682</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Data Clean</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>256</width>
        <height>541</height>
       </rect>
      </property>
      <property name="modelColumn">
       <number>0</number>
      </property>
      <property name="currentRow">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>130</y>
        <width>256</width>
        <height>541</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_search">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>256</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_import_1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>256</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="import_Button">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>120</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>导入实验数据</string>
      </property>
     </widget>
     <widget class="QPushButton" name="input_Button">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>70</y>
        <width>120</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
       </font>
      </property>
      <property name="text">
       <string>导入数据模板</string>
      </property>
     </widget>
     <widget class="QPushButton" name="output_Button">
      <property name="geometry">
       <rect>
        <x>546</x>
        <y>70</y>
        <width>120</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
       </font>
      </property>
      <property name="text">
       <string>导出数据模板</string>
      </property>
     </widget>
     <widget class="QPushButton" name="csvexport_Button">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>380</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>导出CSV</string>
      </property>
     </widget>
     <widget class="QPushButton" name="excelexport_Button">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>470</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>导出Excel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1200</width>
       <height>682</height>
      </rect>
     </property>
     <attribute name="label">
      <string>待续</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
